packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/GroupViewManagerWrapper.kt

```diff
@@ -82,15 +82,12 @@ class GroupViewManagerWrapper(
       }
   }
 
-  override fun getChildAt(parent: ViewGroup, index: Int): View? {
-    viewWrapperDelegate.viewGroupDefinition
+  override fun getChildAt(parent: ViewGroup, index: Int): View {
+    return viewWrapperDelegate.viewGroupDefinition
       ?.getChildAtAction
       ?.let {
-        return it.invoke(parent, index)
-      }
-      .ifNull {
-        return super.getChildAt(parent, index)
-      }
+        return@let it.invoke(parent, index)
```

Comment:
Are you sure that `parent.getChildAt()` can't return a null? What is returned if that child doesn't exist? Is it throwing something in that case?

Reply:
good question! it could [return null](https://developer.android.com/reference/android/view/ViewGroup#getChildAt(int))
> Returns  the view at the specified position or null if the position does not exist within the group

maybe we should report this back to meta.

Reply:
Ok, that's a good idea! We can change that type for now.